watson/utils.py

```diff
@@ -111,3 +113,28 @@ def get_frame_from_argument(watson, arg):
             style('error', "No frame found with id"),
             style('short_id', arg))
         )
+
+
+def get_beginning_arrow(shortcut):
```

Comment:
Can you find a better name for this help function too? Maybe `get_timestamp_for_period` or `get_starttime_for_period`or something similar?